packages/engine/src/worker/runner/rust/behaviors/accessors.rs

```diff
@@ -143,10 +143,9 @@ impl Accessors for Vec3 {
                 unsafe { data_ref.child_data()[0].buffers()[0].typed_data::<f64>() };
             let start_index = i * POSITION_DIM;
             // Does not fail
```

Comment:
Can we either remove these comments or expand on them. I'd prefer an explanation but I don't feel too strongly because we don't have unsafe now (which is a fantastic change)

Reply:
`// Does not fail` could be replaced with a TODO or with the safety comments/cleanup from `iterators.rs`.

Reply:
Replaced by `A position always have 3 values, thus it should never fail`.